Budibase/budibase
# Licensing poc master

## Description
- Cherry pick the commits from https://github.com/Budibase/budibase/pull/4511
- To test against the pre-prod environment 

Also pin sentry node to `6.17.7` due to an issue in `6.17.8`
For more context see: https://github.com/getsentry/sentry-javascript/issues/4594

And update typescript versions to be consistent across all packages
